list_resources.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+SPECROOT=$(pwd)/specs
+for resource in $(ls $SPECROOT); do
+	if [ -d $SPECROOT/$resource ] && [ -n "$(git status -s $SPECROOT/$resource)" ]; then
```

Comment:
What if we delete the resource ?

Reply:
Just ignore the deleted resources? As it makes no sense to delete resources for Ansible and Terraform.

Reply:
It could happen if we need to cleanup the resource folders. Even the case is rare, we should at least make sure it does not fail.

Reply:
I think the script will handle the situation well. It just ignores and does nothing for the deleted resources, and it will not fail even the specs folder is cleaned up.